.github/workflows/build.yaml

```diff
@@ -17,10 +17,15 @@ jobs:
 
     strategy:
       matrix:
-        polyml: ['v5.7.1', 'v5.8.1']
+        polyml: ['v5.7.1', 'v5.9']
```

Comment:
I thought you wanted to remove 5.7.1 and keep 5.8.1 as an alternative to the latest stable version?

Reply:
No, not currently at least. I think it makes more sense to have 5.9 (which is the latest release), and 5.7.1 (which is the latest packaged for LTS Ubuntu). The reason for 5.9 is obvious, I think 5.7.1 makes the most sense as an alternative since this represents people using their default Poly/ML installation. I could be wrong though.

Reply:
Oh yes, sounds very reasonable.